index.html

```diff
@@ -0,0 +1,567 @@
+                            <p>Multimedia</p>
+                            <p>Photography</p>
+                            <p>Video</p>
+                            <p>Newsletters</p>
+                            <p>TimesMachine</p>
+                            <p>NYT Store</p>
+                            <p>Times Journeys</p>
+                            <p>Manage My Account</p>
+                        </div>
+                    </div>
+                    <div class="center-line">
+                        <hr class="v-line">
+                    </div>
+                    <div class="right"> 
+                        <h6 class="title-f">SUSCRIBE</h6>
+                        <p class="right-text"><img class="icon-bottom" alt="home delivery icon" src="https://www.shareicon.net/data/512x512/2015/12/29/694994_delivery_512x512.png"><strong>Home Delivery</strong></p>
+                        <p class="right-text"><img class="icon-bottom" alt="Subscriptions icon" src="https://i.pinimg.com/originals/c4/81/1d/c4811d59c17568b2ea75b1327d0dfc9e.jpg"><strong>Digital Sibscriptions</strong></p>
+                        <p class="right-text"><img class="icon-bottom" alt="crossword icon" src="https://static.thenounproject.com/png/568869-200.png"><strong>Crossword</strong></p>
+                        <p class="right-text"><img class="icon-bottom" alt="cooking icon" src="https://img.pngio.com/clip-art-italy-cooking-png-icon-food-transparent-cartoon-free-cooking-icon-png-920_809.png"><strong>Cooking</strong></p>
+                        <p class="right-text"> Email Newsletters</p>
+                        <p class="right-text"> Education Rate</p>
+                        <hr class="inside-line">
+                        <p class="right-text">Moblie Applications</p>
+                        <p class="right-text">Replica Edition</p>
+                        <p class="right-text">Español</p>
+                        <p class="right-text">中文网</p>
+                    
+                    </div>
+                </div>
+                <hr class="line-bottom">
+            </div>
+            <div class="a11">
+                <p>2020 The New York Times</p>
+                <p>NYTCo</p>
+                <p>Contact Us</p>
+                <p>Work with us</p>
+                <p>Advertise</p>
+                <p>T Brand Studio</p>
+                <p>Your Ad Choices</p>
+                <p>Privacy</p>
+                <p>Terms of Service</p>
+                <p>Site Map</p>
+                <p>Help</p>
+                <p>Subscriptions</p>
+            </div>
+            <br>
+            <br>
+            <br>
+            <br>
+            <div class="a12">
+                <p class="a12-text">Access more of The Times by creating a free account or logging in.</p>
+                <div class="in-a12">
+                    <h6>Expand <i class="fas fa-angle-up"></i></h6>
+                </div>
+            </div>
```

Comment:
- [ ] **OPTIONAL** For semantic HTML, here you can use the `<footer>` tag instead of this `<div>`